### DISCUSSION TITLE
Incorrect default?

### DISCUSSION THREAD

**Discussion by snacks02** [janko/rodauth-rails#242] [General]
I noticed that rodauth-rails defaults the login failure number to `1`:

https://github.com/janko/rodauth-rails/blob/3566cfd09d848b3f2f8ae079268e75813572529c/lib/generators/rodauth/migration/sequel/lockout.erb#L4

Rodauth documentation doesn't seem to specify what the default should be:

![image](https://github.com/janko/rodauth-rails/assets/41530422/c0ad954f-b122-437e-8bfb-6bf884578165)

I also didn't find anything in Rodauth code that would indicate what the correct default value should be.

Is this intended? Shouldn't it default to `0`?

**Comment by janko** [comment]
I was matching the [migrations](https://rodauth.jeremyevans.net/rdoc/files/README_rdoc.html#label-Creating+tables) from the Rodauth README, where the default value of that column is `1` 😉 I suppose that's the default because in case of `0` failures, the record doesn't have to exist at all.

**Comment by snacks02** [comment]
Oh, I see. That makes sense. Thanks for sharing!
